mitosheet/src/mito/utils/actions.tsx

```diff
@@ -1870,6 +1870,86 @@ export const getActions = (
                 windows: 'Ctrl+F'
             }
         },
+        [ActionEnum.OpenSearchAndReplace]: {
+            type: 'build-time',
+            staticType: ActionEnum.OpenSearchAndReplace,
+            longTitle: 'Search and Replace',
+            actionFunction: () => {
+                // We turn off editing mode, if it is on
+                setEditorState(undefined);
+                if (uiState.currOpenSearch.isOpen) {
+                    const searchInput = mitoContainerRef.current?.querySelector<HTMLInputElement>('#mito-search-bar-input');
+                    if (searchInput) {
+                        // If the search bar is already open, then we focus on the input and select all
+                        // to make it easier to search something new without removing the previous search
+                        searchInput.focus();
+                        searchInput.select();
+                    }
+                } else {
```

Comment:
This feels like it shouldn't be in an else. Otherwise, it's not opening replace if the search is already open, ya?

Reply:
Ah yeah true